Personal-Template.

PublisherName opened the pull request:
# Pull Request

## Contribution type

- [x] New template contribution
- [ ] Existing template update
- [ ] Issue fix
- [ ] README.md update

## Contribution Description

[ X ] **I would like my template to be added to the Unstoppable Domains' website builder** 

### Unstoppable Domain Name:

bigrobin.crypto

donald-stolz commented:
@PublisherName Thank you for your submission. Couple of requests for updates

- [ ] Keep all font files local, some off your fonts currently rely on fonts.googleapis.com
- [ ] Inline the `background: url` for the _.main-banner_ class
- [ ] Move `css/`, `fonts/`, and `images/` into a `assets/` folder

I'd also recommend adding social logos or other additional content

PublisherName commented:
Updated:

[x] Keep all font files local, some off your fonts currently rely on fonts.googleapis.com
[x] Inline the background: url for the .main-banner class
[x] Move css/, fonts/, and images/ into a assets/ folder
[x] Added Social Logos.

donald-stolz commented:
Hey thanks for your submission unfortunately we can not accept this template because of W3Layouts [license](https://w3layouts.com/license/). Sorry I had you make updates before looking into this
